azure/template.json

```diff
@@ -138,7 +145,15 @@
     "keyvaultCertificateName": "[if(greater(length(parameters('certificateName')),0), parameters('certificateName'), replace(parameters('customHostName'), '.', '-'))]",
     "appServiceName": "[concat(variables('resourceNamePrefix'), '-as')]",
     "appServicePlanName": "[concat(variables('resourceNamePrefix'), '-asp')]",
-    "appServiceRuntimeStack": "[concat('DOCKER|', parameters('dockerHubUsername'), '/', parameters('containerImageReference'))]"
+    "appServiceRuntimeStack": "[concat('DOCKER|', parameters('dockerHubUsername'), '/', parameters('containerImageReference'))]",
+    "frontDoorName": "[concat(variables('resourceNamePrefix'),'-fd')]",
+    "frontDoorAppServiceName": "[concat(variables('appServiceName'), '.azurewebsites.net')]",
+    "frontDoorWafPolicyName": "[replace(concat(variables('resourceNamePrefix'), '-waf'),'-', '')]",
```

Comment:
WAF policicy names can only have Alphanumeric characters, no '-'s

DAS' convention is suffix of wp for waf policy, it might be ideal to have the same convention

Reply:
frontDoorWafPolicyName is already removing all '-'s
sure, I can change the suffix to wp